cookbook/service_container/scopes.rst

```diff
@@ -96,169 +97,28 @@ when compiling the container. Read the sidebar below for more details.
-
-Now, if you inject this service using setter injection, there are no drawbacks
-and everything works without any special code in your service or in your definition::
-
-    // src/AppBundle/Mail/Mailer.php
-    namespace AppBundle\Mail;
-
-    use AppBundle\Client\ClientConfiguration;
-
-    class Mailer
-    {
-        protected $clientConfiguration;
-
-        public function setClientConfiguration(ClientConfiguration $clientConfiguration = null)
-        {
-            $this->clientConfiguration = $clientConfiguration;
-        }
-
-        public function sendEmail()
-        {
-            if (null === $this->clientConfiguration) {
-                // throw an error?
-            }
-
-            // ... do something using the client configuration here
-        }
-    }
-
-Whenever the ``client`` scope is active, the service container will
-automatically call the ``setClientConfiguration()`` method when the
-``client_configuration`` service is set in the container.
-
-You might have noticed that the ``setClientConfiguration()`` method accepts
-``null`` as a valid value for the ``client_configuration`` argument. That's
-because when leaving the ``client`` scope, the ``client_configuration`` instance
-can be ``null``. Of course, you should take care of this possibility in
-your code. This should also be taken into account when declaring your service:
-
-.. configuration-block::
-
-    .. code-block:: yaml
-
-        # app/config/services.yml
-        services:
-            my_mailer:
-                class: AppBundle\Mail\Mailer
-                calls:
-                    - [setClientConfiguration, ["@?client_configuration="]]
-
-    .. code-block:: xml
-
-        <!-- app/config/services.xml -->
-        <services>
-            <service id="my_mailer"
-                class="AppBundle\Mail\Mailer"
-            >
-                <call method="setClientConfiguration">
-                    <argument
-                        type="service"
-                        id="client_configuration"
-                        on-invalid="null"
-                        strict="false"
-                    />
-                </call>
-            </service>
-        </services>
-
-    .. code-block:: php
-
-        // app/config/services.php
-        use Symfony\Component\DependencyInjection\Definition;
-        use Symfony\Component\DependencyInjection\ContainerInterface;
+.. note::
 
-        $definition = $container->setDefinition(
-            'my_mailer',
-            new Definition('AppBundle\Mail\Mailer')
-        )
-        ->addMethodCall('setClientConfiguration', array(
-            new Reference(
-                'client_configuration',
-                ContainerInterface::NULL_ON_INVALID_REFERENCE,
-                false
-            )
-        ));
+    In Symfony 2.6 and previous versions, there was another alternative based
+    on ``synchronized`` services. However, these kind of services have been
+    removed starting from Symfony 2.7.
```

Comment:
It isn't removed, it's deprecated.

And I'm against making this note unnessecary longer.